guide/C/ch_invest.xml

```diff
@@ -1817,6 +1817,1129 @@ Income
+              use as the total cost of the securities sold will be the same.
+              It is only when you have partially sold a security, that the 
+              costing method is important, so that the Advanced Portfolio Report
+              costs (and therefore gains or losses) are correct for the 
+              remainder of the security holding going forward. As the costing
+              method is a report option, it is preferable that the same costing
+              method is used for all securities. Otherwise, it may be necessary
+              to run multiple Advanced Portfolio Reports, 1 for each costing
+              method and only including the securites using that costing method.
+            </para>
+            
+            <note>
+              <para>Provision has been made in <application>&app;</application>
+                for specifying costing method per security account but this is
+                yet to be implemented.
+              </para>
+            </note>
+        
+            <para>Given that the Advanced Portfolio Report doesn't use any of
+              the lot information, if you want the reported costs and
+              gains/losses to be consistent with the transaction/lot data, it is
+              pointless (other than for the automatic creation of the gain/loss
+              transaction) linking sales with specific acquisitions unless you
+              always sell a particular security LIFO (Last In, First Out). I.e.:
+            </para>
+      
+            <itemizedlist>
+              <listitem>
+                <para>If you always sell <emphasis>FIFO</emphasis>, you can use
+                  the <guilabel>Scrub Account</guilabel> option to scrub the 
+                  oldest security buys and create the gain/loss transaction. Use
+                  the FIFO report option. There is no need to manually link buys
+                  to sell lots. You only really need to scrub if you want 
+                  <application>&app;</application> to calculate and create the 
+                  gain/loss transaction for you.
+                </para>
+              </listitem>
+
+              <listitem>
+                <para>If you always sell <emphasis>LIFO</emphasis>, manually 
+                  create lot(s) for the most recently acquired security buys, 
+                  link them with the sell transaction(s), then scrub specific
+                  lots to create the gain/loss transaction(s). The report should
+                  be correct when you use the LIFO report option.
+                </para>
+              </listitem>
+
+              <listitem>
+                <para> If you cost your sales using <emphasis>Average Costing
+                  </emphasis>, if you scrub, the gain will be calculated using 
+                  the FIFO Cost (as this is the only <application>&app;
+                  </application> method currently implemented), but the report 
```

Comment:
Huh? "cost basis method currently implemented in GnuCash", not "Gnucash method currently implemented".